### PR TITLE
Adding a security reporting procedure

### LINKED ISSUE #7: Security Policy violation SECURITY.md
_This issue was automatically created by [Allstar](https://github.com/ossf/allstar/)._

**Security Policy Violation**
Security policy not enabled.
A SECURITY.md file can give users information about what constitutes a vulnerability and how to report one securely so that information about a bug is not publicly visible. Examples of secure reporting methods include using an issue tracker with private issue support, or encrypted email with a published key.

To fix this, add a SECURITY.md file that explains how to handle vulnerabilities found in your repository. Go to https://github.com/SBOMit/website/security/policy to enable.

For more information, see https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository.

---

This issue will auto resolve when the policy is in compliance.

Issue created by Allstar. See https://github.com/ossf/allstar/ for more information. For questions specific to the repository, please contact the owner or maintainer.

### PR DESCRIPTION
**Pull request by idunbarh**
This is to close #7 and adds a security disclosure process to the website project.

### CODE REVIEW

**Review by trishankatdatadog — COMMENTED** [review]
Should we just link to the security policy [here](https://github.com/SBOMit/specification/security/policy) instead of duplicating? Also, it doesn't make sense to have a security policy for an essentially static website, no?

**Review by JustinCappos — APPROVED** [review]
Looks good

**Review by JustinCappos — APPROVED** [review]

**Comment by JustinCappos** [comment]
Looks like I don't have write access!?!, so we need @jkjell @mnm678 or @colek42 to review before we can merge.

**Review by jkjell — DISMISSED** [review]

**Review by JustinCappos — APPROVED** [review]

**Pull request merged**
